deap/tools/crossover.py

```diff
@@ -5,9 +5,14 @@
 from collections import Sequence
 from itertools import repeat
 
+
 ######################################
 # GA Crossovers                      #
 ######################################
+from deap.gp import Terminal
```

Comment:
@mgard The circular import is caused by HARM being in the GP module. I feel it would be more appropriate if we moved HARM to the algorithm module. Anyhow, I guess this is an other issue and it is fixed by moving the pull request to the gp module anyway.

Reply:
Okay, I'll move it to the gp module